src/styles/blocks/header.scss

```diff
@@ -38,5 +41,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: $transition-time;
```

Comment:
Use the shorthand transition property for better readability. You can combine transition-duration and transition-property like this: transition: color $transition-time.